api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionIrsPaths.scala

```diff
@@ -49,4 +49,23 @@ trait SubmissionIrsPaths
         }
       }
     }
+
+  // institutions/<institutionId>/filings/<period>/submissions/<submissionId>/irs/csv
+  def submissionIrsCsvPath(institutionId: String)(implicit ec: ExecutionContext) =
+    path("filings" / Segment / "submissions" / IntNumber / "irs" / "csv") { (period, seqNr) =>
+      timedGet { uri =>
+        completeVerified(institutionId, period, seqNr, uri) {
+          val validationStats = system.actorSelection("/user/validation-stats")
```

Comment:
Looks like these three lines are duplicated from the other IRS endpoint. I think we should extract these into a method.